src/app/core-ui/main/status/modal/help-modal/console-modal.component.html

```diff
@@ -27,20 +27,20 @@
             </div>
             <div  fxFlex="10" fxLayoutAlign="top top">
               <!-- @Todo Need to change icon -->
-              <mat-icon fontSet="faIcon" fontIcon="fa-mars-stroke-h" *ngIf="list.getType() === 1"></mat-icon>
+              <mat-icon fontSet="faIcon" fontIcon="fa-mars-stroke-h" *ngIf="list.type === 1"></mat-icon>
               <!-- @Todo Need to change icon -->
-              <mat-icon fontSet="faIcon" fontIcon="fa-mars-stroke-v" *ngIf="list.getType() === 2"></mat-icon>
+              <mat-icon fontSet="faIcon" fontIcon="fa-mars-stroke-v" *ngIf="list.type === 2"></mat-icon>
             </div>
             <div  fxFlex="80">
-              <p [ngClass]="{'warn-text': list.getCode() === -1, 'space-text': list.getType() === 2 }">
-                {{list.getText()}}
+              <p [ngClass]="{'warn-text': list.getCode === -1, 'space-text': list.type === 2 }">
```

Comment:
list.getCode === 1?
getCode was removed..

Reply:
yes it is removed.

Reply:
If it's removed, then why is it still in the code? :stuck_out_tongue: